docs/latest/user/tls-cert-manager.md

```diff
@@ -67,7 +67,7 @@ EOF
 
 The `selfSigned` issuer type is namespaced, which means it will use the Issuer resource kind.
 It can only issue certificates for Secrets in the same namespace, here "default."
-We will later see ClusterIssuer used, which can deal with Secrets (and hence Gateways) in any namespace.
+You could also use [ClusterIssuer](https://cert-manager.io/docs/concepts/issuer/), which can be used from any namespace.
```

Comment:
+1

Suggestion: also change line 68 to something like `Here we create `Issuer` kind which is a namespaced resource` or something similar.

Reply:
Maybe I should just change all of them to `ClusterIssuer` and just mention `Issuer` at the end, next to GC.

Reply:
Changed.